scripts/docker_build_push.sh

```diff
@@ -130,14 +130,16 @@ if [[ -n "${BUILD_TARGET}" ]]; then
   TARGET_ARGUMENT="--target ${BUILD_TARGET}"
 fi
 
-CACHE_REF="${REPO_NAME}-cache:${TARGET}-${BUILD_ARG}"
+CACHE_REF="${REPO_NAME}:cache-${TARGET}-${BUILD_ARG}"
```

Comment:
We actually have a `superset-cache` registry setup that's intended to be used here: https://hub.docker.com/repository/docker/apache/superset-cache/general

The reason for this is so that we don't clutter up the "real" registry with a pile of cache images

Reply:
I tried to use this and got the exact same error messages on forks, I agree it's better to use this for caching